terraform/modules/website-e2e-test/variables.tf

```diff
@@ -23,3 +23,7 @@ variable "content_api_url" {
   type        = string
   description = "A URL pointing to the Content API deployment (staging *or* prod) to test against."
 }
+
+data "google_project" "project" {
```

Comment:
we won't need this here. `variables.tf` should only include `variable` blocks.

Reply:
Removed! _(I think this was purely for the [now-removed] IAM grants. 😄 )_